.github/workflows/test-e2e-pr.yml

```
@@ -4,106 +4,225 @@
+      TEST_COMPONENTS: ${{ needs.detect-changed-files.outputs.changed_components }} ${{ needs.detect-changed-files.outputs.manual_components }}
     steps:
+      # 检出代码
       - uses: actions/checkout@v3
+
+      # 设置pnpm
       - name: Setup pnpm
         uses: pnpm/action-setup@v2
 
+      # 设置Node.js环境
       - name: Setup node
         uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)